Library/Homebrew/formula.rb

```diff
@@ -1923,7 +1835,7 @@ def std_meson_args
   # Standard parameters for npm builds.
   #
   # @api public
-  sig { params(prefix: T.any(String, Pathname, FalseClass)).returns(T::Array[String]) }
+  sig { params(prefix: T.any(NilClass, String, Pathname)).returns(T::Array[String]) }
```

Comment:
This impacts homebrew/core formulae by dropping `FalseClass`. If replacing `false -> nil` is intentional, then should have `odeprecated` period as this is public API and documented, i.e.

https://docs.brew.sh/Node-for-Formula-Authors#installing-module-dependencies-locally-with-std_npm_args

Reply:
Not intentional – I'll open a fix